catalyst/rl/db/mongo.py

```diff
@@ -3,6 +3,8 @@
 from catalyst.rl import utils
 from catalyst.rl.core import DBSpec
 
+import safitty
```

Comment:
all libraries imports should be on top of internal catalyst ones

Reply:
fixed!